packages/ui/src/theme/tokens/components/collection.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  BackgroundColorValue,
+  ColorValue,
+  DesignToken,
+} from '../types/designToken';
+
+interface PaginationTokens {
+  current: {
+    color: DesignToken<ColorValue>;
+    backgroundColor: DesignToken<BackgroundColorValue>;
+  };
+  button: {
+    color: DesignToken<ColorValue>;
+    _hover: {
+      backgroundColor: DesignToken<BackgroundColorValue>;
+      color: DesignToken<ColorValue>;
+    };
+    _disabled: {
+      color: DesignToken<ColorValue>;
+    };
+  };
+}
+
+interface StateTokens {
+  color: DesignToken<ColorValue>;
+  backgroundColor: DesignToken<BackgroundColorValue>;
+}
+
+export interface SearchTokens {
+  color: DesignToken<ColorValue>;
+  button: {
+    color: DesignToken<ColorValue>;
+    _disabled: StateTokens;
+    _hover: StateTokens;
+    _focus: StateTokens;
+  };
+}
+
+interface CollectionTokens {
+  pagination: PaginationTokens;
+  search: SearchTokens;
+}
+
+export const collection: CollectionTokens = {
+  pagination: {
+    current: {
```

Comment:
Maybe we should add a note somewhere that we don't want to just import and reference the pagination tokens like 

```typescript
import {pagination} from './pagination'; 
//...
pagination: pagination
```

Because that will break reference. Also that we are explicitly exposing certain tokens and can add more in the future if necessary.